com.unity.render-pipelines.universal/Runtime/Materials/ParticlesUnlit.mat

```diff
@@ -26,12 +26,13 @@ Material:
   m_LightmapFlags: 4
   m_EnableInstancingVariants: 0
   m_DoubleSidedGI: 0
-  m_CustomRenderQueue: 3050
+  m_CustomRenderQueue: 3000
```

Comment:
Maybe this was due to an upgrader, but at this point we should not backport such changes to LTS. 
Please revert these material changes.

Reply:
Removed changed files